finance-api/api-reference.md

```diff
@@ -0,0 +1,282 @@
+Finance API reference
+===
+## About Finance API
+The Finance API fetches information about transactions that are made through Zettle. For example, card payments made with a Zettle card reader.
+
+* [About Finance API](#about-finance-api)
+  * [Base URL](#base-url)
+  * [OAuth scope](#oauth-scope)
+* [Fetch account balance](#fetch-account-balance)
+  * [Parameters](#parameters)
+  * [Responses](#responses)
+* [Fetch account transactions](#fetch-account-transactions)
+  * [Parameters](#parameters)
+  * [Responses](#responses)
+* [Fetch payout information](#fetch-payout-information)
+  * [Parameters](#parameters)
+  * [Responses](#responses)
+* [Examples](#examples)
+  * [Fetch balance for a liquid account](#fetch-balance-for-a-liquid-account)
+  * [Fetch transactions for a liquid account](#fetch-transactions-for-a-preliminary-account)
+  * [Fetch payout information on a specific day](#fetch-payout-information-on-a-specific-day)
+* [Related resources](#related-resources)
+* [Related API reference](#related-api-reference)
+  
+### Base URL
+https://finance.zettle.com
+
+### OAuth Scope
+`READ:FINANCE`
+<!-- For more information on how to get authorisaition for the scope, see See [OAuth2 API](https://github.com/iZettle/api-documentation/blob/master/authorization.adoc) -->
+
+## Fetch account balance
+Returns the balance that is available in a merchant's Zettle preliminary or liquid account at a specific time.
+
+If the balance is zero, it means the following:
+
+* For the preliminary account, the money is moved from the merchant's preliminary account to their liquid account.
+* For the liquid account, the money is paid out from Zettle to the mechant's bank account.
+<!-- Returns accumulated balance at a specific point in time. After a deposit has been made from LIQUID account to merchant’s bank account (daily, monthly or weekly), the balance is usually zero. If not, it will return the amount that is still due in merchant’s Zettle account. -->
```

Comment:
Do we need to mention this? There are so many ways we affect a merchant's balance other than moving from preliminary to liquid and from liquid to their bank account.